.github/workflows/docker-tests.yml

```diff
@@ -0,0 +1,48 @@
+name: e2e test
+on:
+  push:
+    paths:
+      - 'docker/**'
+      - '.github/**'
+  pull_request:
+  # workflow_dispatch will let us manually trigger the workflow from GitHub actions dashboard.
+  # See https://docs.github.com/en/free-pro-team@latest/actions/managing-workflow-runs/manually-running-a-workflow 
+  workflow_dispatch:
+  create:
+
+jobs:
+  build-on-linux:
+    strategy:
+      matrix:
+        docker-version: [19.03]
+    runs-on: ubuntu-latest
+    name: Build and run Docker images
+    steps:
+      - name: Update Packages
+        run: sudo apt-get update -yqq --fix-missing
+      - name: Install Docker
```

Comment:
fwiw I would remove this step, and the matrix as 19.03 is installed by default and installing takes time